tensorboard/webapp/metrics/views/card_renderer/scalar_card_types.ts

```diff
@@ -52,3 +52,16 @@ export interface ScalarCardPoint extends Point {
 }
 
 export type ScalarCardDataSeries = DataSeries<ScalarCardPoint>;
+
+export interface PartialSeries {
+  seriesId: string;
+  points: ScalarCardPoint[];
+}
+
+export interface PartitionedSeries {
+  seriesId: string; // dataset id.
```

Comment:
iiuc, `PartitionedSeries.seriesId` and `PartialSeries.seriesId` are **not interchangeable**.  For `PartialSeries`, the seriesId must always correspond to a runId, while for `PartitionedSeries`, it does not.  e.g.
- `runSelectionMap.get(partialSeries.seriesId)` is ok
- `runSelectionMap.get(partitionedSeries.seriesId)` is not

I'm wondering how we might avoid someone in the future calling a runId-function on a non-runId.  Should this PR rename the `PartialSeries.seriesId` into `runId`?

Reply:
As per refactor PR, changed it to `runId`.